### PR TITLE
docs: (tutorials/restful.md): [translate] Done

### PR DESCRIPTION
**Pull request by DarrenWong**
update

##### Checklist

- [x] documentation is changed or added

##### Affected core subsystem(s)
docs/source/en/tutorials/restful.md

##### Description of change
Translated [https://github.com/eggjs/egg/blob/master/docs/source/zh-cn/tutorials/restful.md](url), thanks for any comments and reviews

### CODE REVIEW

File: docs/source/en/tutorials/restful.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
+title: Build RESTful API
+---
+Web frameworks are widely used for providing interfaces to the client through web services . Let's use an example [CNode Club](https://cnodejs.org/) to show how to build [RESTful](https://en.wikipedia.org/wiki/REST) API using Egg.
```
Comment (dead-horse) [inline docs/source/en/tutorials/restful.md]: Web

Comment (dead-horse) [inline docs/source/en/tutorials/restful.md]: `web services .` => `Web services.`

File: docs/source/en/tutorials/restful.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+    "reply_count": 155,
+    "visit_count": 28176,
+    "create_at": "2016-09-27T07:53:31.872Z",
+  },
+  {
+    "id": "57ea257b3670ca3f44c5beb6",
+    "author_id": "541bf9b9ad60405c1f151a03",
+    "tab": "share",
+    "content": "content",
+    "title": "《一起学 Node.js》彻底重写完毕",
```
Comment (dead-horse) [inline docs/source/en/tutorials/restful.md]: 这个中文随便换一个英文的标题吧

File: docs/source/en/tutorials/restful.md [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+- status code: 200
+- response body:
+
+```json
+{
+  "id": "57ea257b3670ca3f44c5beb6",
+  "author_id": "541bf9b9ad60405c1f151a03",
+  "tab": "share",
+  "content": "content",
+  "title": "《一起学 Node.js》彻底重写完毕",
```
Comment (dead-horse) [inline docs/source/en/tutorials/restful.md]: 同上

File: docs/source/en/tutorials/restful.md (hunk 1 above)
Comment (DarrenWong) [inline docs/source/en/tutorials/restful.md]: Thanks, 我沿用了 一起学 Node.js》彻底重写完毕 ，变成Finished Rewriting of Let's Learning Node.js Together。想问下markdown的JSON内如何加斜体，因为英文书名好像是斜体 我尝试了**不成功，
*Testing*
```json
*Testing*
```

**Pull request merged**